### PR TITLE
[PERFSCALE-2623] Adds 4.15 Large Scale tests

### PR DESCRIPTION
**Pull request by krishvoor**
This PR adds large scale tests for 4.15 and removes the 4.14 large scale testing as it it now GA

### CODE REVIEW

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86-ovnic.yaml [hunk 1]
```diff
@@ -14,60 +14,60 @@ releases:
...
-- as: ocp-qe-perfscale-aws-cni-120ppn-test-252nodes-scale-ovnic
+- as: node-density-cni-252nodes-120ppn
   cron: 0 12 * * 1,3
   steps:
     cluster_profile: aws-perfscale
     env:
       BASE_DOMAIN: perfscale.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "252"
-      COMPUTE_NODE_TYPE: m5.2xlarge
+      COMPUTE_NODE_TYPE: m6.2xlarge
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86-ovnic.yaml:49]: Will this work? isnt it m6[i,a,g]?

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86-ovnic.yaml]: Updated, ptal

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-ovnic-node-density-cni-252nodes-120ppn

**Review by mukrishn — APPROVED** [review]

**Comment by krishvoor** [comment]
The [issue](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/45345/rehearse-45345-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-ovnic-node-density-cni-252nodes-120ppn/1721585988376989696#1:build-log.txt%3A238) seen is not related to the PR in itself:
```
could not run steps: step node-density-cni-252nodes-120ppn failed: "node-density-cni-252nodes-120ppn" 
pre steps failed: "node-density-cni-252nodes-120ppn" pod "node-density-cni-252nodes-120ppn-openshift-qe-move-pods-infra"

failed: could not watch pod: the pod ci-op-5twz2zfk/node-density-cni-252nodes-120ppn-openshift-qe-move-pods-infra failed after 5m31s (failed containers: test): ContainerFailed one or more containers exited
```

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-ovnic-node-density-cni-252nodes-120ppn

**Comment by jtaleric** [comment]
> The [issue](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/45345/rehearse-45345-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-ovnic-node-density-cni-252nodes-120ppn/1721585988376989696#1:build-log.txt%3A238) seen is not related to the PR in itself:
> 
> ```
> could not run steps: step node-density-cni-252nodes-120ppn failed: "node-density-cni-252nodes-120ppn" 
> pre steps failed: "node-density-cni-252nodes-120ppn" pod "node-density-cni-252nodes-120ppn-openshift-qe-move-pods-infra"
> 
> failed: could not watch pod: the pod ci-op-5twz2zfk/node-density-cni-252nodes-120ppn-openshift-qe-move-pods-infra failed after 5m31s (failed containers: test): ContainerFailed one or more containers exited
> ```

Well, it could be due to the scale of the cluster...

**Pull request merged**